csclient/params/params.go

```diff
@@ -239,6 +239,7 @@ type PromulgateRequest struct {
 // See https://github.com/juju/charmstore/blob/v4/docs/API.md#put-idpublish
 type PublishRequest struct {
 	Published bool
+	Resources map[string]int `json:",omitempty"`
```

Comment:
Please comment with what the key-value is -- especially since it's a public property.